### DISCUSSION TITLE
Running from Code Question

### DISCUSSION THREAD

**Discussion by jheizer** [philosowaffle/peloton-to-garmin#488] [General]
Disclaimer that I've never worked with blazor before.  It seems like the bootstrap packages are not being deploy when running when debugging.  I'm sure I am missing something stupid, but failing to see what.  Do you have any tips?

![Screenshot 2023-04-29 10 02 39 PM](https://user-images.githubusercontent.com/1015493/235333787-1b1460d3-9d79-48e1-9a72-eee3d9b77b36.png)

![Screenshot 2023-04-29 10 03 57 PM](https://user-images.githubusercontent.com/1015493/235333793-7cab28ba-d2bf-4f7c-ad11-38bdda130fe1.png)

Otherwise things are running fine.  I can load from peloton and get a garmin auth code via email, just no prompt to enter it because of js errors first.

**Comment by philosowaffle** [comment]
In Visual Studio, right click on the `WebUI/libman.json` file and choose `Restore Client-Side Libraries`.  That should download the needed dependencies.

https://learn.microsoft.com/en-us/aspnet/core/client-side/libman/libman-vs?view=aspnetcore-7.0#restore-files-manually

**Comment by jheizer** [comment]
Bah thanks.  I knew it was something stupidly obvious.
